cypress/integration/fee-fine/lostItemFee.spec.js

```diff
@@ -0,0 +1,42 @@
+import LostItemFeePolicy from '../../support/fragments/circulation/lost-item-fee-policy';
+import SettingsMenu from '../../support/fragments/settingsMenu';
+
+describe('Fee/fine management', () => {
+  const name = 'A_very_specific_name';
+  const editedName = 'An_edited_name';
+  const duration = '6 week(s)';
+  const lostItemChargeFeeFineError = 'Required if there is a possibility of no fee/fine being charged for a lost item';
+  const duplicateError = 'The Lost item fee policy name entered already exists. Please enter a different name.';
+
+  before('Preconditions', () => {
+    cy.loginAsAdmin({ path: SettingsMenu.circulationLostItemFeePolicyPath, waiter: LostItemFeePolicy.waitLoading });
+  });
+
+  it('C5558 Verify that you can create/edit/delete lost item fee policies', () => {
+    // creating
+    LostItemFeePolicy.startAdding();
+    LostItemFeePolicy.fillName(name);
+    LostItemFeePolicy.save();
+    LostItemFeePolicy.checkErrorMessage(lostItemChargeFeeFineError);
+    LostItemFeePolicy.fillDuration('6', 'week(s)');
+    LostItemFeePolicy.save();
+    LostItemFeePolicy.checkAfterSaving(name, duration);
+
+    // editing
+    LostItemFeePolicy.startEditing();
+    LostItemFeePolicy.fillName(editedName);
+    LostItemFeePolicy.save();
+    LostItemFeePolicy.checkAfterSaving(editedName, duration);
+
+    // creating policy with existing name
+    LostItemFeePolicy.startAdding();
+    LostItemFeePolicy.fillName(editedName);
+    LostItemFeePolicy.fillDuration('6', 'week(s)');
```

Comment:
Please pass the parts of the array from one index to another index using duration variable.